From: Martin Klapetek <mklapetek@kde.org>
Date: Wed, 17 Feb 2016 18:44:16 +0000
Subject: Rename the provider files we ship
X-Git-Url: http://quickgit.kde.org/?p=kaccounts-providers.git&a=commitdiff&h=2307d7df60ee8df6467157305a48e1b5bfd924b6
---
Rename the provider files we ship

This is a different fix for 347219 to make Ubuntu happy, but also to
make everyone else happy.

The files will now not overlap anymore, which should fix Ubuntu
packaging. The problem now got moved to the account management UIs,
which we can fix on our side if we need to.

BUG: 347219
---


--- a/providers/google.provider.in
+++ /dev/null
@@ -1,44 +1,1 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<provider id="google">
-  <_name>Google</_name>
-  <!-- Tooltip text appearing over the button to create this type of account -->
-  <_description>Includes GMail, Hangouts, Google Docs, Google+, YouTube and Picasa</_description>
-  <icon>im-google</icon>
-  <translations>kaccounts-providers</translations>
-  <domains>.*google\.com</domains>
 
-  <template>
-    <group name="auth">
-      <setting name="method">oauth2</setting>
-      <setting name="mechanism">web_server</setting>
-      <group name="oauth2">
-        <group name="web_server">
-          <setting name="Host">accounts.google.com</setting>
-          <setting name="AuthPath">o/oauth2/auth?access_type=offline&amp;approval_prompt=force</setting>
-          <setting name="TokenPath">o/oauth2/token</setting>
-          <setting name="RedirectUri">http://localhost/oauth2callback</setting>
-          <!-- HACK: access_type is non standard, but Google requires it in
-               order to return a refresh token -->
-          <setting name="ResponseType">code</setting>
-          <setting name="Scope" type="as">[
-              'https://docs.google.com/feeds/',
-              'https://www.googleapis.com/auth/googletalk',
-              'https://www.googleapis.com/auth/youtube.upload',
-              'https://www.googleapis.com/auth/youtube',
-              'https://www.googleapis.com/auth/userinfo.email',
-              'https://www.googleapis.com/auth/userinfo.profile',
-              'https://picasaweb.google.com/data/',
-              'https://www.googleapis.com/auth/calendar',
-              'https://www.google.com/m8/feeds/',
-              'https://www.googleapis.com/auth/tasks'
-          ]</setting>
-          <setting name="AllowedSchemes" type="as">['https','http']</setting>
-          <setting name="ClientId">876209205844-9e3g6ocf6ejv48dap37et84ks35rdre0.apps.googleusercontent.com</setting>
-          <setting name="ClientSecret">LvV-hkunrdMj4Ijr74rT6kH7</setting>
-          <setting name="ForceClientAuthViaRequestBody" type="b">true</setting>
-        </group>
-      </group>
-    </group>
-  </template>
-</provider>
-

--- /dev/null
+++ b/providers/kde-google.provider.in
@@ -1,1 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<provider id="google">
+  <_name>Google</_name>
+  <!-- Tooltip text appearing over the button to create this type of account -->
+  <_description>Includes GMail, Hangouts, Google Docs, Google+, YouTube and Picasa</_description>
+  <icon>im-google</icon>
+  <translations>kaccounts-providers</translations>
+  <domains>.*google\.com</domains>
 
+  <template>
+    <group name="auth">
+      <setting name="method">oauth2</setting>
+      <setting name="mechanism">web_server</setting>
+      <group name="oauth2">
+        <group name="web_server">
+          <setting name="Host">accounts.google.com</setting>
+          <setting name="AuthPath">o/oauth2/auth?access_type=offline&amp;approval_prompt=force</setting>
+          <setting name="TokenPath">o/oauth2/token</setting>
+          <setting name="RedirectUri">http://localhost/oauth2callback</setting>
+          <!-- HACK: access_type is non standard, but Google requires it in
+               order to return a refresh token -->
+          <setting name="ResponseType">code</setting>
+          <setting name="Scope" type="as">[
+              'https://docs.google.com/feeds/',
+              'https://www.googleapis.com/auth/googletalk',
+              'https://www.googleapis.com/auth/youtube.upload',
+              'https://www.googleapis.com/auth/youtube',
+              'https://www.googleapis.com/auth/userinfo.email',
+              'https://www.googleapis.com/auth/userinfo.profile',
+              'https://picasaweb.google.com/data/',
+              'https://www.googleapis.com/auth/calendar',
+              'https://www.google.com/m8/feeds/',
+              'https://www.googleapis.com/auth/tasks'
+          ]</setting>
+          <setting name="AllowedSchemes" type="as">['https','http']</setting>
+          <setting name="ClientId">876209205844-9e3g6ocf6ejv48dap37et84ks35rdre0.apps.googleusercontent.com</setting>
+          <setting name="ClientSecret">LvV-hkunrdMj4Ijr74rT6kH7</setting>
+          <setting name="ForceClientAuthViaRequestBody" type="b">true</setting>
+        </group>
+      </group>
+    </group>
+  </template>
+</provider>
+

--- /dev/null
+++ b/providers/kde-owncloud.provider.in
@@ -1,1 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<provider id="owncloud">
+  <_name>ownCloud</_name>
+  <icon>owncloud</icon>
+  <!-- Tooltip text appearing over the button to create this type of account -->
+  <_description>ownCloud account</_description>
+  <translations>kaccounts-providers</translations>
+  <plugin>owncloud_plugin_kaccounts</plugin>
 
+    <template>
+        <group name="auth">
+            <setting name="method">password</setting>
+            <setting name="mechanism">password</setting>
+        </group>
+        <group name="dav">
+            <setting name="host"></setting>
+        </group>
+    </template>
+</provider>
+

--- /dev/null
+++ b/providers/kde-twitter.provider.in
@@ -1,1 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<provider id="twitter">
+  <_name>Twitter</_name>
+  <icon>im-twitter</icon>
+  <translations>kaccounts-providers</translations>
+  <domains>.*twitter\.com</domains>
 
+  <template>
+    <group name="auth">
+      <setting name="method">oauth2</setting>
+      <setting name="mechanism">HMAC-SHA1</setting>
+      <group name="oauth2">
+        <group name="HMAC-SHA1">
+          <setting name="ConsumerKey">xBbiAYBmglxmRKBqimLSSw</setting>
+          <setting name="ConsumerSecret">c7QqEjrOVENhGYGFjcUsx6Nwr9GXPLjUZVY2xObrHw8</setting>
+          <setting name="RequestEndpoint">https://api.twitter.com/oauth/request_token</setting>
+          <setting name="TokenEndpoint">https://api.twitter.com/oauth/access_token</setting>
+          <setting name="AuthorizationEndpoint">https://api.twitter.com/oauth/authorize</setting>
+          <setting name="Callback">http://localhost.com/twitter_login.php</setting>
+        </group>
+      </group>
+    </group>
+  </template>
+</provider>
+

--- a/providers/owncloud.provider.in
+++ /dev/null
@@ -1,20 +1,1 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<provider id="owncloud">
-  <_name>ownCloud</_name>
-  <icon>owncloud</icon>
-  <!-- Tooltip text appearing over the button to create this type of account -->
-  <_description>ownCloud account</_description>
-  <translations>kaccounts-providers</translations>
-  <plugin>owncloud_plugin_kaccounts</plugin>
 
-    <template>
-        <group name="auth">
-            <setting name="method">password</setting>
-            <setting name="mechanism">password</setting>
-        </group>
-        <group name="dav">
-            <setting name="host"></setting>
-        </group>
-    </template>
-</provider>
-

--- a/providers/twitter.provider.in
+++ /dev/null
@@ -1,25 +1,1 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<provider id="twitter">
-  <_name>Twitter</_name>
-  <icon>im-twitter</icon>
-  <translations>kaccounts-providers</translations>
-  <domains>.*twitter\.com</domains>
 
-  <template>
-    <group name="auth">
-      <setting name="method">oauth2</setting>
-      <setting name="mechanism">HMAC-SHA1</setting>
-      <group name="oauth2">
-        <group name="HMAC-SHA1">
-          <setting name="ConsumerKey">xBbiAYBmglxmRKBqimLSSw</setting>
-          <setting name="ConsumerSecret">c7QqEjrOVENhGYGFjcUsx6Nwr9GXPLjUZVY2xObrHw8</setting>
-          <setting name="RequestEndpoint">https://api.twitter.com/oauth/request_token</setting>
-          <setting name="TokenEndpoint">https://api.twitter.com/oauth/access_token</setting>
-          <setting name="AuthorizationEndpoint">https://api.twitter.com/oauth/authorize</setting>
-          <setting name="Callback">http://localhost.com/twitter_login.php</setting>
-        </group>
-      </group>
-    </group>
-  </template>
-</provider>
-


